client/app/bundles/course/assessment/submission/reducers/history/questions.js

```diff
@@ -0,0 +1,99 @@
+import actions from '../../constants';
+
+export default function (state = {}, action) {
+  switch (action.type) {
+    case actions.FETCH_SUBMISSION_SUCCESS: {
+      if (action.payload.history) {
+        return {
+          ...state,
+          ...action.payload.history.questions.reduce((obj, question) => {
+            const selected = question.answerIds.slice(0, 3);
```

Comment:
Actually why is this repeated here and `GET_PAST_ANSWERS_SUCCESS`?

Reply:
Suggest to extract the number into some constant to be used by the different actions that need it. So that it is easier to change in the future.

Reply:
actually not really needed here, since the selected attribute can be initialized later on, will just remove it first.